talktorials/11_online_apis/11b_online_docking.ipynb

```diff
@@ -53,6 +53,10 @@
     "   * Manuscript: [_Nucleic Acids Res._ (2010), __38__, W724-31](https://academic.oup.com/nar/article/38/suppl_2/W724/1122840)\n",
```

Comment:
<p><strong>Would it make sense to add a point 4 in Docking methodologies considering rigid ligand and flexible receptor ? </strong></p><p> </p><p>The matching algorithms<strong> (MA)</strong> find molecules that have a high degree of shape similarity to the pockets or binding sites. Programs based on MA have the advantage of speed. </p><p> </p><p>Stochastic methods search the conformational space by randomly modifying a ligand conformation or a population of ligands. Two examples are Monte Carlo <strong>Methods</strong> and <strong>Genetic algorithms.</strong></p><p> </p><p>The Monte Carlo method involves modifying ligands through step-by-step bond rotation, rigid body displacement and rotation. The resulting conformity is tested with an energy-based selection criterion. If it is fulfilled, this conformity is stored and further modified to generate the next conformation until a predefined number <strong>(number of ?)</strong>  is reached. </p><p> </p><p>Knowledge-based scoring functions verwendet statistische Analyse von Kristallstrukturen von Liganden-Protein-Komplexen. Die Idee basiert auf der Annahme, dass je stärker ein Interaktion ist, desto größer ist die Häufigkeit des Auftretens. So können aus den Strukturinformationen, die interatomaren Kontaktfrequenzen zwischen dem Protein und dem Liganden abgeleitet werden<strong>. (Oupsi,.. my German is lacking)</strong></p><p> </p><p><strong><span class="ql-cursor">﻿</span></strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I think a point 4 for rigid ligand and flexible receptor doesn't make sense. I haven't found this case in the literature. As I understood it, one would prefer to consider both as flexible, but to calculate the receptor as flexible is very difficult. Hence the compromise rigid receptor and flexible ligand, instead of both flexible.&nbsp; </p><p> </p><p>Is that correct?</p>